frontend/talentsearch/src/js/components/reviewMyApplication/ReviewMyApplicationPage.tsx

```diff
@@ -0,0 +1,266 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+import UserProfile from "@common/components/UserProfile";
+import Pending from "@common/components/Pending";
+import NotFound from "@common/components/NotFound";
+import { commonMessages } from "@common/messages";
+import { getLocale } from "@common/helpers/localize";
+import MissingSkills from "@common/components/skills/MissingSkills";
+import { notEmpty } from "@common/helpers/util";
+import ExperienceSection from "@common/components/UserProfile/ExperienceSection";
+import { Link } from "@common/components";
+import { getMissingSkills } from "@common/components/skills/MissingSkills/MissingSkills";
+import {
+  Applicant,
+  Maybe,
+  PoolAdvertisement,
+  useGetReviewMyApplicationPageDataQuery,
+} from "../../api/generated";
+import { useDirectIntakeRoutes } from "../../directIntakeRoutes";
+import ApplicationPageWrapper from "../ApplicationPageWrapper/ApplicationPageWrapper";
+import { flattenExperienceSkills } from "../experienceAndSkills/ExperienceAndSkills";
+
+interface ReviewMyApplicationProps {
+  applicant: Applicant;
+  poolAdvertisement: PoolAdvertisement;
+  poolCandidateId: string;
+  closingDate: Date;
+  jobTitle: Maybe<string>;
+  poolId: string;
+}
+
+export const ReviewMyApplication: React.FunctionComponent<
+  ReviewMyApplicationProps
+> = ({
+  applicant,
+  poolAdvertisement,
+  poolCandidateId,
+  closingDate,
+  jobTitle,
+  poolId,
+}) => {
+  const intl = useIntl();
+  const directIntakePaths = useDirectIntakeRoutes();
+  const experiences = applicant.experiences?.filter(notEmpty) || [];
+  const missingSkills = {
+    requiredSkills: poolAdvertisement.essentialSkills?.filter(notEmpty),
+    optionalSkills: poolAdvertisement.nonessentialSkills?.filter(notEmpty),
+  };
+  const hasExperiences = notEmpty(applicant.experiences);
+  const { isProfileComplete } = applicant;
+  const isApplicationComplete =
+    isProfileComplete ||
+    getMissingSkills(
+      missingSkills.requiredSkills || [],
+      hasExperiences ? flattenExperienceSkills(experiences) : [],
+    ).length === 0;
+
```

Comment:
I don't think this works
if `isProfileComplete` is true it never checks after the `||` therefore always returning application as complete regardless of skills provided profile is complete, that is why I think the missing skills check is not working for me

think something like this may work
```
isApplicationComplete = isProfileComplete === true &&  getMissingSkills(
      missingSkills.requiredSkills || [],
      hasExperiences ? flattenExperienceSkills(experiences) : [],
    ).length === 0;
```

Reply:
Fixed by [Fix isApplicationComplete logic](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3796/commits/1ec56f42f5d09e7c9b763c88bef8aabaec7f94ef)